app/crud/crud_data.py

```diff
@@ -16,5 +18,23 @@ def get_data_interval_query(self, db: Session, from_date: datetime.date, to_date
         query = db.query(Data).filter(and_(Data.date <= to_date, Data.date >= from_date, Data.time >= rule_from_time, Data.time <= rule_to_time)).order_by(Data.date.asc(), Data.time.asc()).group_by(Data.id, Data.date)
         return query
 
+    def get_data_interval_query_by_dataset_id(self, db: Session, dataset_id: int) -> Query:
```

Comment:
as I underestand this is not a data interval query

Reply:
Yes! This is poorly named, since it was a interval when I copied it from the upper crud, but since the upper one isn't in use anymore (going to remove it) I'm going to rename this one to get_data_query_by_dataset_id.